src/MongoDB/Manager.c

```diff
@@ -191,6 +195,10 @@ static PHP_METHOD(Manager, __construct)
 	}
 
 	phongo_manager_init(intern, uri_string ? uri_string : PHONGO_MANAGER_URI_DEFAULT, options, driverOptions TSRMLS_CC);
+
+	if (intern->client) {
+		php_phongo_set_monitoring_callbacks(intern->client);
+	}
```

Comment:
In that case, I think we can still move this to `phongo_manager_init()` just to keep this logic out of the constructor. I'd like to revisit down the line with some testing over multiple requests to see if re-assignment is really necessary. Since the callbacks are fixed functions, I have a feeling this has no effect.